cmd/config.go

```diff
@@ -259,3 +260,29 @@ type PortConfig struct {
 	HTTPSPort int
 	TLSPort   int
 }
+
+// BeelineConfig provides config options for the Honeycomb beeline-go library,
+// which are passed to its beeline.Init() method.
+type BeelineConfig struct {
+	// WriteKey is the API key needed to send data Honeycomb. Never check in a
+	// value for this config parameter in a publicly-readable repository.
```

Comment:
Take a look at how we handle secrets for dburls and PKCS#11 PINs. We don't include them in the JSON config directly, but include a pointer to a file from which they can be read. This supports storing the secrets separately from the config, and may also help with the issue you're concerned with here, a dev accidentally committing a test config with a live API key.

Also, I assume Honeycomb supports enough privilege separation that, e.g. a write API key doesn't grant read access, and it would be easy for our dev API keys to not have any privileges related to prod data?

Reply:
Yeah that's a good idea, I can make this `writeKeyFile`, and have the `cmd.BeelineConfig.Get()` load the secret from that file if specified.

Yes, we can create multiple different API keys for the "team", and give each key different sets of permissions.